package.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "x5gon",
+  "version": "0.0.1",
+  "description": "connecting OER repositories",
+  "main": "index.js",
+  "scripts": {
+    "start:server": "cd src/server && node server.js",
+    "start:serverDebug": "cd src/server && node --debug server.js",
```

Comment:
Did you know you can use node --inspect server.js? Then you can connect to your server with Chrome to identify slow parts of code, discover memory leaks, etc. (Adding that comment just because I found it useful in the last few months/weeks.)